java/client/src/main/java/com/youtube/vitess/client/cursor/SimpleCursor.java

```diff
@@ -59,10 +59,7 @@ public boolean next() throws SQLException {
   }
 
   @Override
-  protected Row getCurrentRow() throws SQLException {
-    if (row == null) {
-      throw new SQLDataException("no current row");
-    }
+  public Row getCurrentRow() {
```

Comment:
Yes, I like that idea. It makes sense now that we are no longer matching the JDBC ResultSet style anyway.